screens/Menu/CourseSchedule.js

```diff
@@ -23,53 +24,135 @@ import PropTypes from "prop-types";
+                <View style={styles.menuButtonContainer}>
+                    <TouchableOpacity style={styles.menuButton} onPress={goToMenu}>
+                        <Feather name="menu" style={styles.icon} />
+                    </TouchableOpacity>
+                </View>
+                <Text style={styles.mainLabel}>My Course Schedule</Text>
+                <View style={styles.scrollTextContainer}>
+                    {(switchVal == "true" && calendarEventsList) &&
+                        <FlatList
+                            data={calendarEventsList}
+                            keyExtractor={(item) => item.id}
+                            initialNumToRender={10}
+                            renderItem={({ item }) => (
+                                <ListItem
+                                    title={item.summary}
+                                    titleStyle={{ color: "white", paddingLeft: 20 }}
+                                    subtitle={item.description !== undefined ? item.description : "Please enter a description in your Google Calendar"}
+                                    rightTitleStyle={{ color: "white", paddingRight: 20 }}
+                                    containerStyle={{ backgroundColor: "#2A2E43" }}
+                                    subtitleStyle={{ color: "grey", paddingLeft: 20 }}
+                                    rightIcon={() => item.location &&
+                                        <CheckBox
+                                            size={30}
+                                            iconRight
+                                            iconType='material'
+                                            uncheckedIcon={'arrow-forward'}
+                                            uncheckedColor="#3ACCE1"
+                                            onPress={() => { goToDoubleSearch(item) }
+                                            }
+                                        />}
+                                />
+                            )}
+                            refreshing={refresh}
+                            onRefresh={handleRefresh}
+                        />
+                    }
+                </View>
             </View>
-            <Text style={styles.mainLabel}>My Course Schedule</Text>
-            <SafeAreaView style={styles.scrollTextContainer}>
-                <SectionList
-                    sections={data}
-                    renderItem={({ item }) => <TouchableOpacity onPress={goToCourseSchedule}><Text style={styles.listItem}>{item}</Text></TouchableOpacity>}
-                    renderSectionHeader={({ section }) => <Text style={styles.sectionHeader}>{section.title}</Text>}
-                    keyExtractor={(index) => index}
-                    ItemSeparatorComponent={() => <View style={styles.line} />}
-                />
-            </SafeAreaView>
-
-        </View >
-    );
+        );
+    }
+    if (switchVal == "false") {
```

Comment:
I think it should be `else` and not an if statement because, what if `switchVal` is "_true_" but `CalendarList` is null?

Reply:
You are right ! I will fix this.

Reply:
I changed it !